doc/user-manual/language/polarity.lagda.rst

```diff
@@ -0,0 +1,109 @@
+..
+  ::
+  {-# OPTIONS --polarity #-}
+
+  module language.polarity where
+
+  open import Agda.Builtin.Nat
+
+.. _polarity:
+
+======================
+ Polarity Annotations
```

Comment:
Could you add some links to the relevant literature where polarity is described in more detail?